static/locales/ru.json

```
@@ -1631,13 +1631,15 @@
         "exportSettingsFailed": "Ошибка экспорта настроек: {{error}}",
         "capturingPortrait": "Захват портрета...",
         "generatingCard": "Создание карточки персонажа...",
+        "idleMotionHint": "При сохранении выбранное движение будет установлено как движение ожидания",
```

Biome (2.4.13)

[error] The key idleMotionHint was already declared.

(lint/suspicious/noDuplicateObjectKeys)